Makefile

```diff
@@ -3,12 +3,62 @@
 # Moreover, any project using sandbox as dependency won't build unless
 # there's Foundry installed on the machine.
 
-.PHONY: artifacts
+.PHONY: artifacts cairo-lang local-contracts starkgate-contracts-latest starkgate-contracts-old
+
+CAIRO_LANG_COMMIT_HASH="8e11b8cc65ae1d0959328b1b4a40b92df8b58595"
+STARKGATE_CONTRACTS_COMMIT_HASH="45941888479663ac93e898cd7f8504fa9066c54c"
 
 artifacts:
```

Comment:
should we rename this to piltover and keep only the piltover part?

Reply:
it still has the unsafe proxy stuff

Reply:
Removed.